<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TFColorDisplayClass</class>
 <widget class="QDialog" name="TFColorDisplayClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>真/假彩色显示</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="10" column="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QSpinBox" name="spinBox_3"/>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>蓝色通道波段</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QSpinBox" name="spinBox_2"/>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>绿色通道波段</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QSpinBox" name="spinBox"/>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>红色通道波段</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>选择栅格图层</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
